### PR TITLE
feat: chain id from indexer

### PR DESCRIPTION
**Pull request by mateuszjasiuk**
Also:
- real apr
- real unbonding time
- some fixes to unbonds table

### CODE REVIEW

File: apps/namadillo/src/App/Staking/UnbondingAmountsTable.tsx [hunk 1]
```diff
@@ -24,8 +24,14 @@ export const UnbondingAmountsTable = (): JSX.Element => {
 
     const rowsList: TableRow[] = [];
     for (const myValidator of myUnbonds.data) {
-      const { validator, unbondedAmount } = myValidator;
-      if (myValidator.unbondedAmount?.gt(0)) {
+      const { validator, unbondedAmount, withdrawableAmount } = myValidator;
+
+      const amount =
```
Comment (mateuszjasiuk) [inline apps/namadillo/src/App/Staking/UnbondingAmountsTable.tsx:29]: Not the nicest but the quickest solution for now

File: apps/namadillo/src/slices/validators.ts [hunk 2]
```diff
@@ -216,22 +237,37 @@ const toMyValidators = (
...
       totalVotingPower,
-      unbondingPeriod
+      unbondingPeriod,
+      apr
     );
+    const secondsLeft = BigInt(Number(indexerUnbond.withdrawTime) - timeNow);
+
+    // TODO: later return from the backend
+    const canWithdraw = secondsLeft <= BigInt(0);
+    const timeLeft = timeFromSeconds(canWithdraw ? BigInt(0) : secondsLeft);
```
Comment (pedrorezende) [inline apps/namadillo/src/slices/validators.ts:256]: Maybe we should think about adding a date library like `date-fns` to make things easier. Right now I'm getting the following output `0 Days: 9 Hrs: 18 Mins` , I think it should read something like `9h 18 min`

**Review by pedrorezende — APPROVED** [review]
Few minor comments that we can open issues in the future to fix, if you want!

**Review by jurevans — APPROVED** [review]
Nice, LGTM!

**Pull request merged**